Validation/Geometry/test/dddvsdb/runDDDvsDBGeometryValidation.sh

```diff
@@ -360,9 +360,9 @@ else
 endif
 cd ${myDir}
 
-less $CMSSW_BASE/src/Geometry/CaloEventSetup/test/GeometryCaloValidationDDD.log | tee -a GeometryValidation.log
-less $CMSSW_BASE/src/Geometry/CaloEventSetup/test/GeometryCaloValidationDB.log | tee -a GeometryValidation.log
-less $CMSSW_BASE/src/Geometry/CaloEventSetup/test/GeometryCaloValidationLocal.log | tee -a GeometryValidation.log
+less $CMSSW_BASE/src/Geometry/CaloEventSetup/test/GeometryCaloValidationDDD.log >> GeometryValidation.log
```

Comment:
`#! /bin/tcsh`

Depending on the shell `>> `may fail if the file does not exist.

Reply:
Line 36 guarantees the log file will exist. Then `>>` is used throughout the script after that. And `tcsh` seems to allow `>>` to create a file, so I don't think we need to worry here.